db/migrate/20171011100324_create_user.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUser < ActiveRecord::Migration[5.1]
+  def up
+    create_table :users do |t|
+      t.column :name, :string, unique: true
+      t.column :password_digest, :string
```

Comment:
👍  reversible migration. let's get some `null: false` constraints where appropriate also, across all of your migration files.

Reply:
Done.